src/helpers/stringHelper.js

```diff
@@ -1,31 +1,30 @@
-import * as R from 'ramda';
+import * as R from "ramda";
 
-export const invalidFirstName = (firstName)=> {
-    return (
-        R.isEmpty(firstName) ||
-        R.gt(firstName.length, 32) ||
-        R.lt(firstName.length, 3)
-    );
+export const invalidFirstName = firstName => {
+  return (
+    R.isEmpty(firstName) ||
+    R.gt(firstName.length, 10) ||
+    R.lt(firstName.length, 3)
+  );
 };
-  
-export const invalidLastName = (lastName) => {
-    return (
-       R.isEmpty(lastName) || R.gt(lastName.length, 32) || R.lt(lastName.length, 3)
-    );
+
+export const invalidLastName = lastName => {
+  return (
+    R.isEmpty(lastName) || R.gt(lastName.length, 10) || R.lt(lastName.length, 3)
+  );
 };
-  
-const isValidEmail = (email) => {
-    const regex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
-    return regex.test(email);
+
+const isValidEmail = email => {
+  const regex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+  return regex.test(email);
 };
-  
-export const invalidEmail = (email) => {
-    return R.isEmpty(email) || R.gt(email.length, 32) || !isValidEmail(email);
+
+export const invalidEmail = email => {
+  return R.isEmpty(email) || R.gt(email.length, 32) || !isValidEmail(email);
 };
-  
-export const invalidPassword = (password) => {
-    return (
-       R.isEmpty(password) || R.lt(password.length, 5) || R.gt(password.length, 32)
-    );
+
+export const invalidPassword = password => {
+  return (
+    R.isEmpty(password) || R.lt(password.length, 6) || R.gt(password.length, 10)
```

Comment:
Passwords are also < 10 chars now?

Reply:
Validation matches API. If you want to change the specific validation requirements, lmk.

Reply:
Nope, that's fine.